source/shared/cpp/ObjectModel/Enums.cpp

```diff
@@ -201,7 +201,7 @@ namespace AdaptiveSharedNamespace
             {ActionType::Custom, "Custom"},
             {ActionType::UnknownAction, "UnknownAction"}});
 
-    DEFINE_ADAPTIVECARD_ENUM_DEFAULT(HeightType, HeightType::Stretch, {
+    DEFINE_ADAPTIVECARD_ENUM_DEFAULT(HeightType, HeightType::Auto, {
```

Comment:
This would be a breaking change, no? We should probably talk about this in Design Discussions :)

Reply:
@paulcam206,
Yes, we should talk this about in the design discussion.
I will back out this line of change.